2001-3000/LIT2388Taamme.xml

```diff
@@ -109,7 +108,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
             <relation name="dcterms:isPartOf" active="LIT2388Taamme" passive="LIT2286ZenaSellase"/>
               <relation name="lawd:hasAttestation" active="LIT2388Taamme" passive="IVorlov40"/>
               <relation name="dcterms:isPartOf" active="LIT2388Taamme" passive="LIT2282Trinity" cert="medium"/>
-         </listRelation>
+                       </listRelation>
+                       <desc>The exact composition of and relations between various collections of miracles dedicated
+                         to the Trinity have not been clarified yet.</desc>
```

Comment:
not outside the relation though. abstract addition was fine too.

Reply:
thanks!